cirq/contrib/qasm_import/_parser_test.py

```diff
@@ -303,6 +303,71 @@ def test_u3_angles():
                                                     atol=1e-7)
 
 
+@pytest.mark.parametrize(
+    'expr',
+    [
+        '.333 + 4',
+        '1.0 * 2',
+        '0.1 ^ pi',
+        '0.1 / pi',
+        '2.0e-05 ^ (1/2)',
+        '1.2E+05 * (3 + 2)',
+        '123123.2132312 * cos(pi)',
+        '123123.2132312 * sin(2 * pi)',
+        '3 - 4 * 2',  # precedence of *
+        '3 * 4 + 2',  # precedence of *
+        '3 * 4 ^ 2',  # precedence of ^
+        '3 - 4 ^ 2',  # precedence of ^
+        '3^2^(-2)',  # right associativity of ^
+        '(-1) * pi',
+        '(+1) * pi',
+        '-3 * 5 + 2',
+        '(+4 * (-3) ^ 5 - 2)',
+        'tan(123123.2132312)',
+        'ln(pi)',
+        'exp(2*pi)',
+        'sqrt(4)',
+        'acos(1)',
+        'atan(0.2)',
+    ])
+def test_expressions(expr: str):
+    qasm = """OPENQASM 2.0;
+     qreg q[1];
+     U({}, 2 * pi, pi / 2.0) q[0];
+""".format(expr)
+
+    parser = QasmParser()
+
+    q0 = cirq.NamedQubit('q_0')
+
+    expected_circuit = Circuit()
+    expected_circuit.append(
+        QasmUGate(float(sympy.sympify(expr)) / np.pi, 2.0, 1 / 2.0)(q0))
+
+    parsed_qasm = parser.parse(qasm)
+
+    assert parsed_qasm.supportedFormat is True
```

Comment:
Unless this value can be something besides true or false, just assert it instead of asserting is true